content/docs/components/serving/seldon.md

```diff
@@ -6,72 +6,18 @@ weight = 40
 
 ## Serve a model using Seldon
 
-_This section has not yet been converted to kustomize, please refer to [kubeflow/manifests/issues/10](https://github.com/kubeflow/manifests/issues/10)._
+Seldon comes installed with Kubeflow.
 
-[Seldon-core](https://github.com/SeldonIO/seldon-core) provides deployment for any machine learning runtime that can be [packaged in a Docker container](https://docs.seldon.io/projects/seldon-core/en/latest/wrappers/README.html).
+The documentation for using Seldon can be found [here](https://docs.seldon.io/projects/seldon-core/en/latest/).
 
-Install the seldon package:
+If you have a saved model on a PVC, Google Bucket or AWS S3 you can use one of the prepacked servers as discussed [here](https://docs.seldon.io/projects/seldon-core/en/latest/servers/overview.html).
 
-```
-ks pkg install kubeflow/seldon
-```
-Generate the core components for v1alpha2 of Seldon's CRD:
+Seldon also provides language specific wrappers to wrap your inference code for it to run in Seldon as discussed [here](https://docs.seldon.io/projects/seldon-core/en/latest/wrappers/README.html).
```

Comment:
For clarity and accessibility (e.g. in screen readers) it's best to give more information than just `here` in link alias text. 

Suggestion:

```
... to run in Seldon. See the [Seldon docs](https://docs.seldon.io/projects/seldon-core/en/latest/wrappers/README.html).
```